UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/ShowAnnualNMs.cshtml

```diff
@@ -1,4 +1,7 @@
-﻿<h4>
+﻿@using UKHO.MaritimeSafetyInformation.Common.Models.NoticesToMariners
+@model ShowWeeklyFilesResponseModel
+
```

Comment:
Please delete this unused [ShowAnnualNMs.cshtml](https://github.com/UKHO/Maritime-Safety-Information/pull/82/files#diff-ec110a17c3d4447d463cc7b95883bd18a59118543f406494b9e850876325fe52) file with static content as is not required now.

Reply:
Done